test/assertions.js

```diff
@@ -59,9 +59,9 @@ exports.mixinMochaAssertions = function(expect) {
       }
     )
     .addAssertion(
-      '<RawRunResult|JSONRunResult> [not] to have completed with [exit] code <number>',
+      '<RawResult|RawRunResult|JSONRunResult> [not] to have [completed with] [exit] code <number>',
       function(expect, result, code) {
```

Comment:
why those additional "[ ... ]"?

Reply:
This rule was duplicated above, so I consolidated it.

So this allows you to say "<var> to have completed with exit code <code>" or "<var> to have exit code <code>" or "<var> to have completed with code <code>".

I find it's nicer to provide several different ways of expressing the same thing.